inc/class-talk-settings-page.php

```diff
@@ -95,6 +107,24 @@ public function render_base_url_field() {
 		<?php
 	}
 
+	/**
+	 * Prints input field for static url setting.
+	 *
+	 * @since 0.1.0
+	 */
+	public function render_static_url_field() {
```

Comment:
Should fields include a note explaining the difference between `coral_talk_base_url` and `coral_talk_static_url`?

Reply:
Added